### ISSUE TITLE
"First run" mail can not be transmitted

### ISSUE THREAD

**Issue by silicht** [containrrr/watchtower#156]
Hi,

I have an error while transmitting "First run" mail via google smtp servers:

```
550 This message contains malformed MIME (demime acl condition: base64 line length exceeds 76 characters).
```

This is the content of the failing mail:

```
Return-Path: <ACCOUNT@gmail.com>
Received: from localhost (mail.DOMAIN. [IP])
        by smtp.gmail.com with ESMTPSA id ID
        for <MY_MAIL@ADDRESS>
        (version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
        Tue, 16 Jan 2018 04:14:59 -0800 (PST)
Message-ID: <ID@mx.google.com>
Date: Tue, 16 Jan 2018 04:14:59 -0800 (PST)
From: ACCOUNT@gmail.com
X-Google-Original-From: watchtower@DOMAIN
To: MY_MAIL@ADDRESS
Subject: Watchtower updates on 20fa564825b1
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: base64

MjAxOC0wMS0xNiAxMjoxNDo1OCAoaW5mbyk6IEZpcnN0IHJ1bjogMjAxOC0wMS0xNiAxMjoxOTo1
OCArMDAwMCBVVEMNCg==
```

Has anybody the same issue?

Thanks!

**Comment by stffabi** [comment]
Thanks for your report. Yes this seems to be a problem right now and could also happen for the other messages. It depends on the length of the message and if the mail sever checks for the line length.

**Comment by unki** [comment]
I'm encountering a similar issue - in my case, when `watchtower` wants to notify that it has updated some containers. Mailserver `Exim` (http://exim.org/) rejects the message submission in SMTP chat and `watchtower` then logs:

```
Failed to send notification email:  550 maximum allowed line length is 998 octets, got 1292
```

**Comment by stffabi** [comment]
Fixed with `v0.2.2`
